core/clients/blobberClient/client.go

```diff
@@ -0,0 +1,173 @@
+package blobberClient
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net"
+	"net/url"
+
+	"github.com/0chain/gosdk/core/encryption"
+
+	"github.com/0chain/blobber/code/go/0chain.net/blobbercore/convert"
+
+	blobbercommon "github.com/0chain/blobber/code/go/0chain.net/core/common"
+	"github.com/0chain/gosdk/zboxcore/client"
+	"google.golang.org/grpc/metadata"
+
+	"github.com/0chain/blobber/code/go/0chain.net/blobbercore/blobbergrpc"
+	"google.golang.org/grpc"
+)
+
+const GRPCPort = 7031
```

Comment:
This probably requires a design discussion

Reply:
We are currently using a fixed grpc port, this should be configurable by the blobber. Blobber should provide the grpc addr its listening on, currently it provides only http addr - this is a large task and is best handled in another issue. https://github.com/0chain/gosdk/issues/52